lumigator/jobs/evaluator/schemas.py

```diff
@@ -17,7 +17,7 @@ class ModelConfig(BaseModel):
 
 
 class EvaluationConfig(BaseModel):
-    metrics: list[str] = Field(default=["rouge", "meteor", "bertscore"])
+    metrics: list[str] = Field(default=["rouge", "meteor", "bertscore", "bleu"])
```

Comment:
We should at some point think about how we should decide what metrics to use by default or whether we should always run all them by default, but that's a question for another day :)